src/components/AppLayout/Sidebar/index.tsx

```diff
@@ -76,8 +77,13 @@ const Sidebar = ({
         <List items={items} />
       </>
     ) : null}
-
     <HelpContainer>
+      {!IS_PRODUCTION && safeAddress && (
+        <>
+          <StyledDivider />
+          <DevTools />
```

Comment:
I would also lazy import it if possible, so that it's not imported on prod.

Reply:
> On my screen, it makes the sidebar quite cramped. I would place these tools in a new menubar on top (or bottom) of the app. Along with the CGW toggle

I think we'll quickly run out of space with extra features regardless of if we use the sidebar of an appbar. I feel like there's more wasted space in the sidebar (on my resolution at least).

I'd like to look into moving the devtools into a hideable popup, much like [this](https://react-hook-form.com/dev-tools/).